src/mempoollimit.h

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2019 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php .
+
+#ifndef MEMPOOLLIMIT_H
+#define MEMPOOLLIMIT_H
+
+#include <map>
+#include <set>
+#include <vector>
+
+#include "boost/optional.hpp"
+#include "primitives/transaction.h"
+#include "uint256.h"
+
+const size_t DEFAULT_MEMPOOL_TOTAL_WEIGHT_LIMIT = 80000000;
+const int64_t DEFAULT_MEMPOOL_EVICTION_MEMORY_MINUTES = 60;
```

Comment:
I don't understand the considerations that went into this figure.

Reply:
They're described in the draft ZIP.